region/lowernorfair/west.json

```diff
@@ -68,13 +67,31 @@
           "to": [
             {
               "id": 3,
-              "requires": null,
-              "unlock": null,
-              "note": "FIXME Some acid damage will be required without certain items"
+              "requires": [
```

Comment:
I intended for the node to just represent being at the ledge. Clearing the PB blocks and interacting with the statue was meant to be part of the node's `unlock` reqs.

I'm a bit disappointed I didn't leave a note on the node to explain that.

Reply:
I interpret leave-as-is